lib/ipaddress/ipv4.rb

```diff
@@ -805,6 +805,42 @@ def subnet(subprefix)
       end
     end
 
+    #
+    # Subtract other from the current range
+    # Returns list of ranges that excludes :other
+    #
+    # Example:
+    #
+    #   a = IPAddress '10.0.0.0/16'
+    #   b = IPAddress '10.0.128.0/18'
+    #   c = IPAddress '192.168.0.0/16'
+    #
+    #   a.subtract(b)
+    #   => 
+    #   [#<IPAddress::IPv4:0x0000000111e61a88 @address="10.0.0.0", @allocator=0, @octets=[10, 0, 0, 0], @prefix=18, @u32=167772160>,
```

Comment:
These lines are long, is there another way you can illustrate the output, without using the string returned by inspect?

Reply:
That's a common practice for tests code. Also, it's very demonstrative. I don't see issues having these lines.